src/playwright-api/makeTest.ts

```diff
@@ -38,7 +61,10 @@ export const makeTest = (
         const completeArchive = await createResourceArchive(page);
         await use();
 
-        const sourceMap = await takeArchive(page, testInfo);
+        let sourceMap;
+        if (!disableAutoCapture) {
```

Comment:
So, if we disable the auto capture, what happens in this case? It seems like it's not actually going to do anything, but I could be wrong?

Reply:
That's correct!

Reply:
Ok, I'm not sure I'm understanding. If someone disables auto capture, what's the point of even running `test-archiver` then? Sorry if I'm being naive.

Reply:
Oh, good point! We should probably just exit from the fixture immediately. I'll give that a try.